service/lib/agama/software/callbacks/signature.rb

```diff
@@ -56,6 +56,11 @@ def setup
         # @param repo_id [Integer] Repository ID. It might be -1 if there is not an associated repo.
         def accept_unsigned_file(filename, repo_id)
           repo = Yast::Pkg.SourceGeneralData(repo_id)
+
+          # Temporarily disable signature checking for local repositories.
+          # https://github.com/agama-project/agama/issues/2092
+          return true if repo && repo["url"].start_with?("dir://")
```

Comment:
Libzypp allows `dir:/directory` URL (with a single slash), I'd make the test less strict.

```suggestion
          return true if repo && repo["url"].start_with?("dir:/")
```

Reply:
Yes, I tested it locally. I have changed it to `dir:`, which is the scheme, and that's it.